commit 6f15b178cd6315c997981f76c6ebed7ad39144c5
Author: Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>
Date:   Thu Oct 5 15:42:42 2023 +0530

    EDAC/versal: Add a Xilinx Versal memory controller driver
    
    Add a EDAC driver for the RAS capabilities on the Xilinx integrated DDR
    Memory Controllers (DDRMCs) which support both DDR4 and LPDDR4/4X memory
    interfaces. It has four programmable Network-on-Chip (NoC) interface
    ports and is designed to handle multiple streams of traffic. The driver
    reports correctable and uncorrectable errors, and also creates debugfs
    entries for testing through error injection.
    
      [ bp:
       - Add a pointer to the documentation about the register unlock code.
       - Squash in a fix for a Smatch static checker issue as reported by
         Dan Carpenter:
         https://lore.kernel.org/r/a4db6f93-8e5f-4d55-a7b8-b5a987d48a58@moroto.mountain
      ]
    
    Co-developed-by: Sai Krishna Potthuri <sai.krishna.potthuri@amd.com>
    Signed-off-by: Sai Krishna Potthuri <sai.krishna.potthuri@amd.com>
    Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>
    Signed-off-by: Borislav Petkov (AMD) <bp@alien8.de>
    Link: https://lore.kernel.org/r/20231005101242.14621-3-shubhrajyoti.datta@amd.com

diff --git a/MAINTAINERS b/MAINTAINERS
index 35977b269d5e..628d02fa549a 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -23719,6 +23719,13 @@ F:	Documentation/devicetree/bindings/media/xilinx/
 F:	drivers/media/platform/xilinx/
 F:	include/uapi/linux/xilinx-v4l2-controls.h
 
+XILINX VERSAL EDAC DRIVER
+M:	Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>
+M:	Sai Krishna Potthuri <sai.krishna.potthuri@amd.com>
+S:	Maintained
+F:	Documentation/devicetree/bindings/memory-controllers/xlnx,versal-ddrmc-edac.yaml
+F:	drivers/edac/versal_edac.c
+
 XILINX WATCHDOG DRIVER
 M:	Srinivas Neeli <srinivas.neeli@amd.com>
 R:	Shubhrajyoti Datta <shubhrajyoti.datta@amd.com>